### PR TITLE
Force SSL on deployed IM

### PR DESCRIPTION
**Pull request by jbzdak**
Only thing to review here is 22a6d0f882760878d0aee3528e6481f986341ea4 . Rest was reviewed by me as part in #68

### CODE REVIEW

File: deploy/requirements.yml [hunk 1]
```diff
@@ -41,4 +41,4 @@
 
 - name: opencraft
   src: https://github.com/open-craft/ansible-opencraft
-  version: 'origin/smarnach/opencraft'
+  version: 'origin/jbzdak/ssl-rebased'
```
Comment (e-kolpakov) [inline deploy/requirements.yml]: @jbzdak not sure if it is a problem or not, but shouldn't we use master here?

Comment (jbzdak) [inline deploy/requirements.yml]: For now we plan to use commit hashes, but only for things that are merged to master, if you approve this one: https://github.com/open-craft/ansible-opencraft/pull/3 I'll update this to hash :)

File: deploy/requirements.yml [hunk 2]
```diff
@@ -41,4 +41,4 @@
 
 - name: opencraft
   src: https://github.com/open-craft/ansible-opencraft
-  version: 'origin/smarnach/opencraft'
+  version: a3e9cde2968e9de94cb9e908244d05654def8dcd
```
Comment (e-kolpakov) [inline deploy/requirements.yml]: @jbzdak same note - shouldn't it be `origin/master`?

**Comment by e-kolpakov** [comment]
@jbzdak needs rebasing

File: deploy/requirements.yml (hunk 2 above)
Comment (jbzdak) [inline deploy/requirements.yml]: @e-kolpakov In most of the other repositories I pinned requirements to specific hash, just to have versioning. Does is make sense?

Comment (e-kolpakov) [inline deploy/requirements.yml]: @jbzdak yes, but than it might make sense to actually do versioning, i.e. using git tags. Does it make sense?

Comment (jbzdak) [inline deploy/requirements.yml]: @e-kolpakov I'm fine adding git tags for this purpose. I'll do this later today.

**Comment by e-kolpakov** [comment]
@jbzdak :+1:

**Pull request merged**